src/plugins/expression/expression-tile.tsx

```diff
@@ -65,6 +71,13 @@ export const ExpressionToolComponent: React.FC<ITileProps> = observer((props) =>
     onUnregisterTileApi
   });
 
+  const mathfieldAttributes = {
```

Comment:
@scytacki and @kswenson I made a dynamic set of attributes to spread into a single declared `<math-field>` as we talked about, but still had to instantiate a separate `ref` for the read-only version - without it we lose focus on the editing version after typing a numerator of a fraction.  I am not entirely understanding why, but I think it is related to the library wanting to do various operations on the front end (e.g. change the LaTeX string).  If those changes are triggered on both sides, the focus model is off and one has to refocus manually.

Reply:
You'd also have to change the useEffect code above so it doesn't set the position if it is readOnly. If that doesn't work, this is very confusing.

Reply:
Thanks, this catalyzed a series of changes that also allowed me to only have the one `ref`.  The only thing that was throwing the selection/focus out of whack was the cursor repositioning.